src/main/java/com/minecolonies/coremod/colony/buildings/BuildingMiner.java

```diff
@@ -35,7 +37,7 @@
 /**
  * The miners building.
  */
-public class BuildingMiner extends AbstractBuildingWorker
+public class BuildingMiner extends AbstractBuildingStructureBuilder
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Override the "equals" method in this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2160)